packages/ui-react/src/components/Header/HeaderActionButton.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import IconButton, { type Props as IconButtonProps } from '../IconButton/IconButton';
+
+import styles from './Header.module.scss';
+
+const HeaderActionButton = ({ children, className, ...rest }: IconButtonProps) => {
```

Comment:
Do you think we need this component? Looks like the only difference is in the additional class.

Reply:
This is needed for component composition. Because this uses the `styles.iconButton` from the `Header.module.scss` file, it would be weird to import the header styles instead.

It's not much, but it does make it clear